include/MeshAmalgamationBase.h

```diff
@@ -0,0 +1,46 @@
+#include <cstdlib>
+#include <vector>
+#include <stdexcept>
+
+#include "libmesh/dof_map.h"
+#include "libmesh/elem.h"
+#include "libmesh/equation_systems.h"
+#include "libmesh/libmesh.h"
+#include "libmesh/linear_implicit_system.h"
+#include "libmesh/mesh.h"
+#include "libmesh/numeric_vector.h"
+#include "libmesh/system.h"
+
+using namespace libMesh;
+
+class MeshAmalgamation {
+
+private:
+protected:
+  std::string _variable_name;
+  unsigned int _extra_element_integer_index;
+  unsigned int _variable_index;
+  std::string _system_name;
+  unsigned int _number_of_colors= 12;
+
+  libMesh::Mesh &_mesh;
```

Comment:
Since you have set the namespace, do you need to specify `libMesh::` here?

Reply:
Alternatively, is you want to continue to be clear about the namespace, you can avoid the `using namespace` command